test/run.el

```diff
@@ -108,6 +108,11 @@ Node `(ert)Test Selectors' for information about test selectors."
   (let* ((load-prefer-newer t)
          (source-directory (locate-dominating-file flycheck-runner-file "Cask"))
          (pkg-rel-dir (format ".cask/%s/elpa" emacs-version)))
+
+    ;; standardise on the C locale so we can be sure that neither of
+    ;; gcc and gfortran will output smartquotes
+    (setenv "LC_ALL" "C")
```

Comment:
Is that enough?  Shouldn't we set `$LANG` and `$LC_MESSAGES` as well?  Sorry if that's a stupid question, but I'm not at all familiar with the semantics of all these locale variables.